indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -519,3 +519,43 @@ th.i-table {
         }
     }
 }
+
+#track-list {
+    list-style-type: none;
+    padding-left: 0;
+    width: 800px;
+
+    .i-box {
```

Comment:
Out of curiosity. What's the need for customizing the `.i-box`, `.i-box-header`, and `.i-box-content`?